codegenerator/cli/templates/static/codegen/src/Index.bs.js

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env node
```

Comment:
@DZakh maybe you can also include a comment at the top of this file to explain why it exists.